content/pillow/concepts/image/terms/rotate/rotate.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.rotate()'
+Description: 'Rotates the image around its center anticlockwise by the specified number of degrees.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Computer Vision'
+  - 'Images'
+  - 'Pillow'
+CatalogContent:
+ - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+In Pillow, the **`.rotate()`** method is used to change the rotation of an image. Pillow is a widely used Python library for image processing and manipulation, supporting formats such as JPEG, PNG, PPM, TIFF, GIF and BMP. It can be easily installed on a local PC using [pip](https://www.codecademy.com/resources/docs/python/pip).
```

Comment:
```suggestion
In Pillow, the **`.rotate()`** method is used to rotate an image by a specified number of degrees counterclockwise. Pillow is a popular Python library for image processing and manipulation, with support for formats like JPEG, PNG, PPM, TIFF, GIF, and BMP. It can be easily installed on a local PC using [pip](https://www.codecademy.com/resources/docs/python/pip).
```

Reply:
Just added the "counterclockwise" line here, and updated a line in the center about formats supported since it was plagiarised.

Reply:
> Just added the "counterclockwise" line here, and updated a line in the center about formats supported since it was plagiarised.

I based the entry on the latest pillow term that was on the docs website (resize) as they were quite similar and shared quite a bit of information. It was not my intention to plagiarize, I thought it would be more uniformed and could be easier to compare and contrast the features if they were displayed similarly. I apologize.